src/Kodify/BlogBundle/Controller/PostsController.php

```diff
@@ -61,4 +91,27 @@ public function createAction(Request $request)
 
         return $this->render('KodifyBlogBundle:Default:create.html.twig', $parameters);
     }
+
+    /**
+     * Rates a post
+     *
+     * @param $id post
+     * @param $star number of stars
+     *
+     * @return \Symfony\Component\HttpFoundation\RedirectResponse
+     */
+    public function rateAction($id, $star)
+    {
+        if ($this->getDoctrine()->getRepository("KodifyBlogBundle:Post")->setRate($id, $star)) {
```

Comment:
Hey, you chose to put this method on the repository instead of on the Entity itself, could you explain us why?